x-pack/plugins/apm/ftr_e2e/cypress/integration/read_only_user/service_overview/header_filters.spec.ts

```diff
@@ -144,7 +139,7 @@ describe('Service overview - header filters', () => {
         .find('li')
         .first()
         .click();
-      cy.get('[data-test-subj="suggestionContainer"]').realPress('{enter}');
+      cy.get('[data-test-subj="headerFilterKuerybar"]').type('{enter}');
```

Comment:
what's the difference between .type and .realPress?

Reply:
Cypress events, like `type` or `click`, are simulated, they are fired from Javascript. The real events fire the native event. It's not clear to me though why the `.realPress` stopped working and the `.type` now works though.

Reply:
ok, can you add that as a comment to the flaky tests issue? we can try to fix it there if possible but if we can't no biggie.